launchable/commands/subset.py

```diff
@@ -505,8 +508,19 @@ def run(self):
             else:
                 try:
                     test_runner = context.invoked_subcommand
+
+                    strategy = default_retry_strategy()
```

Comment:
I checked split-subset logic in Launchable server, but it doesn't load the model in that API. Hence, I believe it's enough in only subset API.